scripts/obtain/uninstall/debian-packages.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
I don't think we should have one script for OSX and Ubuntu. 

The processes are meaningfully different and having that detection logic seems unnecessary when a user could very easily choose a different script. 

the name change sounds find though.